### PR TITLE
Outline design of research systems

### PR DESCRIPTION
**Pull request by alice-i-cecile**
> In *Emergence*, there are three distinct forms of research: [assimilation](assimilation.md) (capturing new species), [genetics](genetics.md) (refining and branching existing species) and [hive mind upgrades](hive-mind-upgrades.md) (increasing information and direct player upgrades).

These are my initial thoughts on how research systems should work in *Emergence*. Please do bikeshed me on names!

### CODE REVIEW

**Review by TimJentzsch — CHANGES_REQUESTED** [review]
Very interesting stuff!
I think the assimilation will be the most difficult part from the game design perspective. Especially keeping it fun instead of annoying in subsequent playthroughs when you already know all species.

I left a couple smaller comments for typos and the glossary.

File: design/src/research/hive-mind-upgrades.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+Several categories of upgrade belong here:
+
+- basic upgrades
+  - these are too important to [early game](../glossary.md#early-game-mid-game-and-late-game) [quality of life](../glossary.md#quality-of-life) and [tutorialization](../glossary.md#tutorialization) to be gated behind all but the most basic achievements
+  - alerts
+  - undo/redo
+  - pipette
+  - clipboard
+  - basic encyclopedia
+  - zoning strength
```
[the remark this answers is not in the document]
Comment (alice-i-cecile) [reply design/src/research/hive-mind-upgrades.md:30]: Yeah I need to write up the zoning doc and link to this.

File: design/src/research/hive-mind-upgrades.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+However, simply excluding these tools makes managing late-game factories effectively very hard, and will frustrate genre veterans.
+
+To deal with this tension, many of these tools are traditionally buried in menu settings, hidden behind keybinds or relegated to mods.
+However, there's a better approach: enable them slowly over time in a [diagetic](../glossary.md#diagetic) way.
+
+In *Emergence*, the player controls the hive as a whole.
+We can play into this theme by framing quality of life and information upgrades as the hive mind getting smarter.
+
+To differentiate these further from [genetics](genetics.md) research, these upgrades don't cost resources.
+Instead, they unlock as various [achievements](../glossary.md#achievements) are completed for each playthrough.
```
[the remark this answers is not in the document]
Comment (alice-i-cecile) [reply design/src/research/hive-mind-upgrades.md:14]: Good call!

File: design/src/research/genetics.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# Genetics
+
+**Genetics** allows the player to modify species that are [part of their hive](assimilation.md).
+In this menu, players can:
+
+1. Unlock **mutations** for a species.
+   1. These can add new options for the species.
+   2. These can also dramatically improve the species in some axis to open up a new role for it.
```
[the remark this answers is not in the document]
Comment (alice-i-cecile) [reply design/src/research/genetics.md:8]: Strongly agree!

[the remark this answers is not in the document]
Comment (alice-i-cecile) [reply design/src/research/genetics.md:8]: I would probably call these "double-edged".

**Comment by alice-i-cecile** [comment]
Agreed on the importance of populating glossary entries, but I'm going to split that into another PR :)

**Pull request merged**